bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-options/component.jsx

```diff
@@ -245,13 +274,14 @@ class UserOptions extends PureComponent {
           />
         )
         : null),
-      (isUserModerator
+      (isUserModerator && CaptionsService.isCaptionsEnabled()
```

Comment:
Is the moderator check really needed in this whole component? For this pull request I would not recommend any changes regarding this aspect but maybe we could open a new issue to refactor it. At the moment it's label says "Manage users" but looks like it's deviating from that a little bit

Reply:
Likely we can extract out the check for moderator. In general the entire menu should only be visible for moderators. In terms of managing users -- It's more of "manage users and features" but I would be OK leaving the name as is